tests/verilog/genblk_wire.ys

```diff
@@ -0,0 +1,16 @@
+#logger -expect warning "Identifier `\\genblk1[0].x' is implicitly declared." 1
+#logger -expect warning "Identifier `\\genblk1[1].x' is implicitly declared." 1
```

Comment:
Why are these commented out?

Should they be the following instead?
```
logger -expect warning "Identifier `\\genblk1\[0\]\.x' is implicitly declared\." 1
logger -expect warning "Identifier `\\genblk1\[1\]\.x' is implicitly declared\." 1
```

Reply:
I couldn't get the regex working at the time. I'll try those and update accordingly.

Reply:
This form doesn't work and returns an error:
```
-- Executing script file `genblk_wire.ys' --
Added regex 'Identifier `\\genblk1\[0\]\.x' is implicitly declared\.' for warnings to expected warning list.
ERROR: Error in regex expression 'Identifier `\\genblk1\[0\]\.x' is implicitly declared\.' !
```

Reply:
According to https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_03_05
```
.[\
    The <period>, <left-square-bracket>, and <backslash> shall be special except when used in a bracket expression (see [RE Bracket Expression](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap09.html#tag_09_03_05)). An expression containing a '[' that is unescaped and is not part of a bracket expression produces undefined results.
```
Hence only the left square bracket needs to be escaped, not the right one. Will fix accordingly.